pkg/machines/components/vmnetworktab.jsx

```diff
@@ -155,11 +155,15 @@ class VmNetworkTab extends React.Component {
             },
             {
                 name: _("Source"), value: (network, networkId) => {
-                    const setSourceClass = (source) => checkDeviceAviability(source) ? "machines-network-source-link link-button" : undefined;
+                    const getSourceClass = (source) => checkDeviceAviability(source) ? "machines-network-source-link link-button" : undefined;
+                    const sourceElem = (source) => {
+                        const sourceClass = getSourceClass(source);
+                        return sourceClass ? <button role="link" className={sourceClass} onClick={sourceJump(source)}>{source}</button> : source;
+                    };
```

Comment:
Why use that uneccessary helper function when we can have logic in just single line?
```
const sourceElem = (source) => {
    return checkDeviceAviability(source) ? <button role="link" className="machines-network-source-link link-button" onClick={sourceJump(source)}>{source}</button> : source;
};
```